mp4parse/src/lib.rs

```diff
@@ -2229,6 +2253,27 @@ fn read_iprp<T: Read>(
                                 )?;
                             }
                         }
+                        // XXX this is contrary to the published specification; see doc comment
```

Comment:
This is far enough from the doc comment on `read_iprp` that I'd consider explicitly saying 'see doc comment on read_iprp' or similar.